pytorch_transformers/modeling_roberta.py

```diff
@@ -46,22 +46,46 @@
 }
 
 
-class RobertaEmbeddings(BertEmbeddings):
+class RobertaEmbeddings(nn.Module):
     """
     Same as BertEmbeddings with a tiny tweak for positional embeddings indexing.
     """
     def __init__(self, config):
-        super(RobertaEmbeddings, self).__init__(config)
-        self.padding_idx = 1
+        super(RobertaEmbeddings, self).__init__()
+        self.padding_idx = config.padding_idx
```

Comment:
Just a reminder this value will need to be updated in the roberta config jsons

Reply:
Thank you for your comment :) Yeah, It needs to be considered.